src/js/pm-image-editor.js

```diff
@@ -444,6 +448,11 @@
                     $scope.$broadcast('updateSelection', $scope.editorId, $scope.editor.selectionCss());
 
                     $scope.updateHistoryButtons();
+
+                    $scope.state = angular.copy($scope.editor.history.current());
+                    setTimeout(function() {
+                        $scope.$apply();
+                    }, 0);
```

Comment:
If a $digest/$apply is already in progress, why do you need another outside of the current execution?